packages/hint-http-cache/src/_locales/en/messages.json

```diff
@@ -5,42 +5,42 @@
     },
     "directiveInvalid": {
         "description": "Report message when the directives are invalid",
-        "message": "The $1 $2 $3 invalid"
+        "message": "A 'cache-control' header contains invalid directives: $1"
```

Comment:
Should we wrap the directives with `'`  or `"` for consistency and make it even clearer?

Reply:
Were you thinking of quoting the directives as a group or individually (e.g. when there's more than one)?

Reply:
I think group should be fine as they are part of the same header value (and it should be easier to implement)? Don't have a strong opinion though.

Reply:
Ok, I went with quoting the individual values.